### PR TITLE
[BRE-101] Remove dept-devops from CODEOWNERS

### PR DESCRIPTION
**Pull request by michalchecinski**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/BRE-101

## 🚧 Type of change

-   🎂 Other

## 📔 Objective

Remove dept-devops from all CODEOWNERS, and add dept-bre to `.github/workflows/publish-to-galaxy.yml`.

Remove dept-devops from CODEOWNERS.

## 📋 Code changes

-   **.github/CODEOWNERS:** Remove dept-devops from all CODEOWNERS, and add dept-bre to `.github/workflows/publish-to-galaxy.yml`.

## ⏰ Reminders before review

-   Contributor guidelines followed
-   All formatters and local linters executed and passed
-   Written new unit and / or integration tests where applicable
-   Protected functional changes with optionality (feature flags)
-   Used internationalization (i18n) for all UI strings
-   CI builds passed
-   Communicated to DevOps any deployment requirements
-   Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

-   👍 (`:+1:`) or similar for great changes
-   📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
-   ❓ (`:question:`) for questions
-   🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
    issue and could potentially benefit from discussion
-   🎨 (`:art:`) for suggestions / improvements
-   ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
-   🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
-   ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Review by coltonhurst — DISMISSED** [review]

File: .github/CODEOWNERS
Comment (vgrassia) [inline .github/CODEOWNERS:10]:
```suggestion
## BRE team owns these workflows ##
```

**Review by vgrassia — CHANGES_REQUESTED** [review]
Before this is merged in we need a TSD ticket opened to have `@bitwarden/dept-bre` added to this repository.

**Comment by michalchecinski** [comment]
TSD ticket created: https://bitwarden.atlassian.net/servicedesk/customer/portal/5/TSD-931

**Review by coltonhurst — COMMENTED** [review]
It says the codeowners file contains errors?

**Review by vgrassia — APPROVED** [review]

**Review by coltonhurst — APPROVED** [review]

**Pull request merged**
